components/style/sequential.rs

```diff
@@ -17,12 +17,17 @@ pub fn traverse_dom<N, C>(root: N,
               C: DomTraversalContext<N>
     {
         debug_assert!(context.should_process(node));
-        context.process_preorder(node);
+        let should_stop = match context.process_preorder(node) {
+            ForceTraversalStop::Force => true,
+            ForceTraversalStop::DontForce => false,
```

Comment:
Let's call these Let's call this RestyleResult::{StopTraversal,ContinueTraversal}?